force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +524,16 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if(person1.getAge() == person2.getAge()){
+            return person1;
+        }
+
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
+        }
+        else {
+            return person2; // Replace null with the variable you used to store the result    
+        }
+        
```

Comment:
If there is a tie you can return either. Can you simplify this to one if condition and a return statement.?